content/docs/getting-started-tutorial/set-up.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Getting set up
+subtitle: Learn how to create a new Bigquery project and generate warehouse credentials.
+priority: 0
+---
+
+Welcome to the Dataform Getting Started Tutorial. This tutorial is for people who are new to Dataform and want to be taught how to set up a new project. We will show you how to create your own data model, how to test and document it and how to run schedules on it. You will need a working knowledge of SQL for this tutorial.
+
+For this tutorial we’re going to pretend we are a fictional e-commerce shop. We already have 3 main data sources in our data warehouse:
+
+- Information about our customers coming from Salesforce
+- Orders information from Shopify
+- Payment information from Stripe
+
+The aim of this tutorial is to create two new tables in our warehouse one called `order_stats` and one called `customers`, which are:
+
+- Updated every hour
+- Tested for data quality
+- Well documented
+
+<div style="position: relative; padding-bottom: 55.78124999999999%; height: 0;"><iframe src="https://www.loom.com/embed/2368b67928ec43b2a7eaf8fabda636f9" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
+
+## Create a new Bigquery project
+
+Dataform connects to [many different warehouses](https://docs.dataform.co/warehouses#__next) but for this tutorial we’ll use Bigquery, since anyone with a Google Account can use it and it has a free tier. We’ve created a public dataset in Bigquery that anyone can access for the purpose of this tutorial.
+
+1. First you will need to create a new Bigquery project:
+
+- Go to the [Bigquery console](https://console.cloud.google.com/bigquery) (If you don’t already have a GCP account you’ll need to create one here).
+- If you’ve just created a new account you’ll be asked to create a new project straight away. If you already have an existing account you can select the project drop down in the header bar, and create a new project from there.
+
+<img src="https://assets.dataform.co/getting%20started%20tutorial/set%20up/Screenshot%202020-08-13%20at%2015.40%201%20(1).png" max-width="753"  alt="" />
+
+## Generate warehouse credentials
+
+In order for Dataform to connect to your Bigquery warehouse you’ll need to generate some credentials. Dataform will connect to BigQuery using a service account. You’ll need to create a service account from your Google Cloud Console and assign it permissions to access BigQuery.
+
+1. To create a new service account in Google Cloud Console you need to:
```

Comment:
I think this looks kinda weird, as the bullet points are on the same indentation as the number. Maybe indent them? This is true in a bunch of other places as well.

<img width="629" alt="Screenshot 2020-08-14 at 16 04 43" src="https://user-images.githubusercontent.com/5724925/90263714-f3a5e000-de47-11ea-8a54-5960edf435ad.png">